src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,311 @@
+        regularWidth
+    />
+    <Divider />
+</React.Fragment>
+
+# Behaviors
 
-<!-- @TODO switch icon-navigation links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+## Nested Navigation
 
-<DemoCard
-    repository={'icon-navigation'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
-    react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
+When using drawer navigation, you can have multiple levels of the navigation hierarchy. These appear as expandable panels in the drawer. When using hierarchical navigation in the drawer, you should try to limit yourself to no more than two levels.
+
+<ImageGrid images={[NestedItemImg]} caption={'Open and closed nested navigation.'} regularWidth />
+
+## Content-Cover vs. Content-Resize
+
+When a drawer is opened, there are two ways to deal with the main application content.
+
+With content-resize, the main application content resizes its area to accommodate the expanded drawer. This is the default behavior for most drawer variations.
+
+With content-cover, the navigation panel appears on top of the main application content, partially obscuring it. This typically uses a dark overlay behind the drawer to direct the user's focus to the navigation panel and block interaction with the main content. This pattern is most common when using the temporary drawer variant on mobile and tablet applications.
+
+<React.Fragment>
+    <ImageGrid
+        images={[ContentCoverImg]}
+        caption={[
+            '1. Displays the normal collapsed state of the menu.',
+            '2. Displays the open state of the menu with a gray overlay.',
+        ]}
+        regularWidth
+    />
+    <ImageGrid
+        images={[ContentResizeImg]}
+        caption={[
+            '1. Displays the normal collapsed state of the menu.',
+            '2. Displays the open state of the menu with content pushed to the right.',
+        ]}
+        regularWidth
+    />
+</React.Fragment>
+
+## Dividers
+
+Divider lines can be used between navigation items in the drawer. They are most effective when used for:
+
+-   separating nested drawer items
+-   separating sections of the drawer contents
+
+<React.Fragment>
+    <ImageGrid
+        images={[DividerGroupImage]}
+        regularWidth
+        caption={['Dividers can be used to separate different sections of the drawer contents.']}
+    />
+    <ImageGrid
+        images={[DividerImage]}
+        regularWidth
+        caption={[
+            '1. Standard drawer navigation does not include dividers.',
+            '2. Avoid using divider lines between every navigation item. These can make the drawer appear busy.',
+        ]}
+    />
+    <Divider />
+</React.Fragment>
+
+# Design Specifications
+
+## Mobile
+
+<ImageGrid
+    images={[
+        PanelSpecsMobileImg,
+        NestedItemStandardsImg,
```

Comment:
I don't remember, is the blue background of the drawer primary[50] as listed, or were we doing primary[500] with an opacity filter?

Reply:
These specs also don't match our current implementation of the drawer. We may want to speed up our look into making those style changes.

Reply:
Yeah, in that default drawer discussion we decided the 5% opacity works better. Blue50 is a bit too heavy.